Dockerfile

```diff
@@ -0,0 +1,90 @@
+# base image
+FROM ubuntu:focal
```

Comment:
Hello @mikihiroikura @arrenglover ! Just a curiosity, there is any reason you need to use Ubuntu 20.04 instead of a more modern distribution? We are discussing about dropping Ubuntu 20.04 support for apt dependencies in https://github.com/robotology/robotology-superbuild/issues/1439, and I am afraid that the next YARP release will not support Ubuntu 20.04 with apt dependencies.